ci/use_wheels_from_prs.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2025-2026, NVIDIA CORPORATION.
+# SPDX-License-Identifier: Apache-2.0
+
+# initialize PIP_CONSTRAINT
+source rapids-init-pip
```

Shellcheck (0.11.0)

[info] Not following: rapids-init-pip was not specified as input (see shellcheck -x).

(SC1091)